canonical-web-and-design/jaas-dashboard
# Show the model status using real data

## Done

Show the model status details on the model details page. 

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click on a model you own and it should show your model status.
- Navigate back to the model listing and click on a model you don't own. It should also show you the model status.

## Details

- Requires #70 to land first (contains code from #70).
- This is unstyled, that will be able to be done as a follow-up.
